### PR TITLE
Improvements to style

### PR DESCRIPTION
**Pull request by alexandersundin**

### CODE REVIEW

**Comment by davidhesselbom** [comment]
@simonmika, can you take a look at this?

File: base.css [hunk 1]
```diff
@@ -49,7 +49,8 @@ p {
 	margin: 0;
 }
 p+p {
-	text-indent: 3mm;
+	/*text-indent: 3mm;*/
```
Comment (simonmika) [inline base.css]: Use this.

File: base.css [hunk 2]
```diff
@@ -49,7 +49,8 @@ p {
...
+	padding-top: 1em;
```
Comment (simonmika) [inline base.css]: Not this.

**Comment by alexandersundin** [comment]
Updated

**Pull request merged**